test/integration/testsuite/minishift.go

```diff
@@ -350,3 +350,26 @@ func (m *Minishift) minishiftHomeDirectoryShouldntExist(dir string) error {
 
 	return fmt.Errorf("Directory %s exists", dir)
 }
+
+func (m *Minishift) setEnvironmentVariable(variable string, value string) error {
+	os.Setenv(variable, value)
+	if os.Getenv(variable) != value {
+		return fmt.Errorf("Error in setting environment variable %s=%s in the current shell", variable, value)
+	}
+	return nil
+}
+
+func (m *Minishift) getEnvironmentVariable(variable string, value string) error {
+	if os.Getenv(variable) != value {
```

Comment:
Here I suggest to first retrieve the value, check for error and then compare:
```
actualValue, err := os.Getenv(variable)
if err != nil {
  return err
}

if actualValue != expectedValue {
  return fmt.Errorf("Retrieved environmental variable does not match expected value. Expected: '%s', Actual: '%s'", expectedValue, actualValue)
}
```

Reply:
@agajdosi I think this won't work as ```os.Getenv(variable)``` returns string otherwise empty if key is not present. See https://golang.org/pkg/os/#Getenv